content/blog/2020-08-05-cml-self-hosted-runners-on-demand-with-gpus.md

```diff
@@ -0,0 +1,198 @@
+---
+title: CML self-hosted runners on demand with GPUs
+date: 2020-08-05
+description: |
+  A tutorial on how to setup your own self-hosted runners with GPUs using CML cloud-runners in your CML workflow in Github or Gitlab.
+picture: 2020-08-05/header.png
+author: david_g_ortega
+commentsUrl: https://discuss.dvc.org/t/cml-self-hosted-runners-on-demand-with-gpus/??
```

Comment:
we can go and create the topic I guess //cc @elleobrien

Reply:
Yep I'll open it now

Reply:
https://discuss.dvc.org/t/cml-self-hosted-runners-on-demand-with-gpus/462

Reply:
👍